src/z3c/relationfield/README.rst

```diff
@@ -874,3 +874,12 @@ It's pointing to the nonexistent path:
 
   >>> root['e'].rel.to_path
   'nonexistent'
+
+Setting up a releation catalog
+==============================
+
+This package provides a RelationCatalog initialized with a set of indexes commonly useful for queries on RelationValue objects.
+The default indexes are `from_id`, `to_id`, `from_attribute`, `from_interfaces_flattened` and `to_interfaces_flattened`.
+
+However, sometimes it is needed to define custom indexes.
+The `zc.relationfield.index.RelationCatalog` class can be initialized with a list of dicts with keys `name` and `kwargs` to be passed to RelationCatalog to be passed to the catalogs `addValueIndex` method.
```

Comment:
Typo: catalogs -> catalog's.

Also, I don't think `name` is passed to addValueIndex, rather it's used to look up a field in the IRelationValue interface.  That should be documented.

It also implies a limitation that you cannot define indexes that aren't part of the IRelationValue interface -- well, you can subclass or call addValueIndex by hand.  In other ways, the main purpose of this change is not to define additional custom indexes but to skip defining one of the default ones, AFAIU?

Reply:
Well, yes. It could be extended to use additional fields. At first the idea was to change the use of interfaces classes to dotted paths, but this would break existing catalogues or need migrations on those. This way we can get rid of the two indexes. So, I suppose the correct way would be to pass the looked up Element instead of the name in order to allow more flexible usage?

Reply:
> So, I suppose the correct way would be to pass the looked up Element instead of the name in order to allow more flexible usage?

That sounds good!  You'd also rename `name` to `element` then?